Package.swift

```diff
@@ -16,7 +16,7 @@ if ProcessInfo.processInfo.environment["IS_XCTEST"] != nil ||
 let package = Package(
     name: "EmbraceIO",
     platforms: [
-        .iOS(.v13), .tvOS(.v13), .macOS(.v13), .watchOS(.v6)
+        .iOS(.v16), .tvOS(.v13), .macOS(.v13), .watchOS(.v6)
```

Comment:
I assume this means the Benchmark stuff is only available in iOS 16+?
Not sure we can actually go through with this change...